GNUmakefile

```diff
@@ -64,7 +64,12 @@ terraform build provider:
 mmv1:
 	cd mmv1;\
 		bundle; \
-		bundle exec compiler.rb -e terraform -o $(OUTPUT_PATH) -v $(VERSION) $(mmv1_compile);
+		if [ "$(VERSION)" = "ga" ]; then \
```

Comment:
I assume that after this change, we'll generate beta docs for GA even when building providers locally? I think this is probably preferable as it's consistent with public providers?

Reply:
correct !